src/components/Main.js

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
- [ ] You did great job so far and it is a nice effort to write unit tests for the MonsterCard component and for the monsters reducer function. But it would make your testing complete if you add tests for the MainMonstersList component in the Main.js. As it is a component connected to the store you need to use `redux-mock-store` library. Kindly add this test and you are good to go.